### DISCUSSION TITLE
Defer multiple public responses to slash commands

### DISCUSSION THREAD

**Discussion by tbezierslafosse** [kordlib/kord#910] [Help] [answered]
Hi,

I've been recently using Kord and it's awesome, thanks a lot for the great work.
I have a question though, I set up slash commands in my discord server, and in some cases the answer my bot writes are longer than 2000 characters (the limit for free discord accounts).

I chunked my long answers in several strings, but I can't defer multiple responses to my command. 
Any idea on how to do it ? 

Thanks :)

**Comment by DRSchlaubi** [comment]
After deferring you can use follow-ups, deferring just tells discord that you received the interaction and want to respond in >3s

**Reply by tbezierslafosse** [reply]
Alright, I'll try this asap. Thanks a lot :)

**Reply by lukellmann** [reply, marked as answer]
the code would look something like this:
```kt
val deferred = interaction.deferPublicResponse()
val response = deferred.respond { content = "initial response" }
response.createPublicFollowup { content = "followup 1" }
response.createPublicFollowup { content = "followup 2" }
```
